.travis/check_coverage_file.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Simple check if 'coverage.md' was updated with PR.
+
+set -euv
+
+# skip this check for everything but PRs
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  exit 0
+fi
+
+# skip if '[nocoverage]' found
+if [ "$TRAVIS_COMMIT_RANGE" != "" ]; then
+  RANGE=$TRAVIS_COMMIT_RANGE
+elif [ "$TRAVIS_COMMIT" != "" ]; then
+  RANGE=$TRAVIS_COMMIT
+fi
+
+# Travis sends the ranges with 3 dots. Git only wants two.
+if [[ "$RANGE" == *...* ]]; then
+  RANGE=`echo $TRAVIS_COMMIT_RANGE | sed 's/\.\.\./../'`
+else
+  RANGE="$RANGE~..$RANGE"
+fi
+
+if [[ $(git log --format=medium --no-merges "$RANGE" | grep "\[nocoverage\]") ]]
+then
+  echo "[nocoverage] is present - skipping check"
+  exit 0
+fi
+
+coverage_file_name="coverage.md"
+coverage_original_file_name="coverage_original.md"
+master_version="https://raw.githubusercontent.com/pulp/pulp_rpm/master/${coverage_file_name}"
```

Comment:
Is this being used?

Reply:
they are, coverage* for diff check, master version for get original file

Reply:
It looks like you're curling `https://raw.githubusercontent.com/pulp/pulp_rpm/master/CHANGES.rst` though?